deploy/single_server/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
         release: {{ .Release.Name }}
 
     spec:
+      securityContext:
```

Comment:
What is the difference in having securityContext here compared to below. Also, why is runAsUser not moved as well?

Reply:
securityContext should be optional (running it as userId 1000 throws exceptions while container starts).
tested with v1.20.2 of k8s the position of securityContext should be placed at spec level